lib/rules/accessor-pairs.js

```diff
@@ -1,5 +1,5 @@
 /**
- * @fileoverview Rule to flag wrapping non-iife in parens
+ * @fileoverview Requires to have a getter for every property which has a setter defined.
```

Comment:
Nice catch! Can we have this match the description in the rule metadata, for consistency?

Reply:
@kaicataldo Sure!